zebra-state/tests/basic.rs

```diff
@@ -7,24 +7,28 @@ use zebra_test::transcript::Transcript;
 
 use zebra_state::*;
 
-static ADD_BLOCK_TRANSCRIPT: Lazy<Vec<(Request, Response)>> = Lazy::new(|| {
-    let block: Arc<_> =
-        Block::zcash_deserialize(&zebra_test::vectors::BLOCK_MAINNET_415000_BYTES[..])
-            .unwrap()
-            .into();
-    let hash = block.as_ref().into();
-    vec![
-        (
-            Request::AddBlock {
-                block: block.clone(),
-            },
-            Response::Added { hash },
-        ),
-        (Request::GetBlock { hash }, Response::Block { block }),
-    ]
-});
+type Error = Box<dyn std::error::Error + Send + Sync + 'static>;
+type ErrorChecker = fn(Error) -> Result<(), Error>;
```

Comment:
We will probably want to export this alias in `zebra_test`, or maybe make it an associated type alias with the `Transcript`.

Reply:
Both aliases or just one?

Reply:
Just the one seems best.